source/adminguide/networking/isolation_in_advanced_zone_with_vlan.rst

```diff
@@ -80,6 +80,11 @@ Prerequisites
 Creating a PVLAN-Enabled Network
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+   .. note::
+      If you are facing issues with connectivity in PVLANS, especially community and promiscuous PVLANS
+      in a multi hypervisor environment, this could be caused by the way PVLANs were implemented
```

Comment:
Could be easier to mention `community and promiscuous PVLANs were introduced after 4.14.0 and were not supported on previous versions` rather than `this could be caused by the way PVLANs were implemented in 4.14.0, which was later fixed in 4.15.0`?

Reply:
It was introduced for VMware in 4.14.0 which is what could be causing the problem, hence stating that it is fixed now
But I'll be happy to change it if it sounds too confusing